library/pk_wrap.c

```diff
@@ -882,7 +881,7 @@ static int ecdsa_sign_wrap( void *ctx_arg, mbedtls_md_type_t md_alg,
 
     /* mbedtls_pk_write_key_der() expects a full PK context;
      * re-construct one to make it happy */
-    key.pk_info = &pk_info;
+    key.pk_info = &mbedtls_eckey_info;
```

Comment:
Note: copy-pasta in the commit message. Can you rewrite history to fix it?

Reply:
Yep will rewrite